floris/turbine_library/iea_22MW.yaml

```diff
@@ -0,0 +1,178 @@
+# Data based on:
+# https://github.com/IEAWindSystems/IEA-22-280-RWT/blob/main/Documentation/
+# IEA-22-280-RWT_tabular.xlsx
+turbine_type: 'iea_22MW'
+hub_height: 170.0
+rotor_diameter: 284.0
+TSR: 9.15
+operation_model: cosine-loss
+power_thrust_table:
+  ref_air_density: 1.225
+  ref_tilt: 6.0
+  cosine_loss_exponent_yaw: 1.88
+  cosine_loss_exponent_tilt: 1.88
+  power:
+    - 0.000000
+    - 0.000000
+    - 375.538649
+    - 635.644519
+    - 1014.237244
+    - 1473.497284
+    - 1996.315118
+    - 2571.170922
+    - 3179.901342
+    - 3803.380108
+    - 4421.435886
+    - 5013.265683
+    - 5559.92078
+    - 6043.266596
+    - 6446.694086
+    - 6756.659289
+    - 6962.663346
+    - 7057.781739
+    - 7076.914762
+    - 7211.728426
+    - 7466.186181
+    - 7847.828144
+    - 8367.545594
+    - 9039.659135
+    - 9882.192126
+    - 10918.83042
+    - 12176.33054
+    - 13684.38108
+    - 15971.52541
+    - 18121.24994
+    - 20200.10629
+    - 22000.0
+    - 21999.98265
+    - 21999.98772
+    - 22000.00215
+    - 22000.0002
+    - 21999.90199
+    - 22000.08648
+    - 22000.0445
+    - 22000.0247
+    - 22000.01503
+    - 22000.00988
+    - 22000.00673
+    - 22000.00464
+    - 22000.00305
+    - 22000.00173
```

Comment:
Recommend you shave off these decimal places above rated, or at least make monotonic, there is a slight wiggle which messes up some of the optimizers